config/settings/review.yml

```diff
@@ -7,6 +7,7 @@ environment:
 
 publish_url: https://qa.publish-teacher-training-courses.service.gov.uk
 find_url: https://qa.find-postgraduate-teacher-training.service.gov.uk
+extra_find_url: https://qa.find-teacher-training-courses.service.gov.uk
```

Comment:
This is probably not used since the review domain is set to https://find-review-4299.test.teacherservices.cloud/

Reply:
With the simplified implementation I remove the extra_find_url. So now is good